sentinel-core/src/test/java/com/alibaba/csp/sentinel/util/function/Tuple2Test.java

```diff
@@ -0,0 +1,14 @@
+package com.alibaba.csp.sentinel.util.function;
```

Comment:
Could you please add the license header here?

Reply:
done